suite-common/token-definitions/package.json

```diff
@@ -25,10 +25,11 @@
         "@trezor/connect": "workspace:*",
         "@trezor/env-utils": "workspace:*",
         "@trezor/type-utils": "workspace:*",
-        "@trezor/utils": "workspace:*",
-        "jws": "^4.0.0"
+        "@trezor/utils": "workspace:*"
     },
     "devDependencies": {
-        "ajv": "^8.17.1"
+        "@trezor/eslint": "workspace:*",
+        "ajv": "^8.17.1",
+        "jws": "^4.0.0"
```

Comment:
`jws` can be removed completely, since you're exporting encode / decode from `@suite-common/suite-utils`, no? 🤔

Reply:
`cmd + shift + f` betrayed me 🙈